clang/lib/CIR/FrontendAction/CIRGenAction.cpp

```diff
@@ -62,15 +62,17 @@ class CIRGenConsumer : public clang::ASTConsumer {
   IntrusiveRefCntPtr<llvm::vfs::FileSystem> FS;
   std::unique_ptr<CIRGenerator> Gen;
   const FrontendOptions &FEOptions;
+  CodeGenOptions &codeGenOptions;
```

Comment:
```suggestion
  CodeGenOptions &CodeGenOptions;
```
Keep clang-style identifier naming throughout this file.